src/Lib.hs

```diff
@@ -15,24 +21,24 @@ import Data.Maybe (fromJust)
 
 
 chordToNotes :: Chord -> [Root]
-chordToNotes chord@(Chord root _ _ _ _) = 
-  (flip jumpIntervalFromNote root) <$> (S.toList $ chordToIntervals chord)
+chordToNotes chord@(Chord root _ _ _ _) =
+  flip jumpIntervalFromNote root <$> S.toList (chordToIntervals chord)
 
 
-chordToIntervals :: Chord -> Set Interval 
+chordToIntervals :: Chord -> Set Interval
 chordToIntervals (Chord root quality highNat exts sus) =
   let
-    baseScale = highestNaturalToIntervals highNat $ qualityToIntervals quality  
+    baseScale = highestNaturalToIntervals highNat $ qualityToIntervals quality
     intervals = susIntervals (extendIntervals baseScale exts) sus
-  in 
-    foldr (\int -> \intSet -> S.insert int intSet) S.empty intervals
+  in
+    foldr S.insert S.empty intervals
```

Comment:
Notice that in the old version, we're taking two arguments in a particular order and applying `S.insert` to them in that same order; this means that we've added an unnecessary extra layer of function(s) around `S.insert`, when it already has the correct type for `foldr`.

Reply:
N.B. This is _also_ an eta-reduction, as above, just manifest a little bit differently (the lambda is explicit here.)